service/src/main/java/bio/terra/workspace/db/WorkspaceActivityLogDao.java

```diff
@@ -37,18 +51,27 @@ public WorkspaceActivityLogDao(NamedParameterJdbcTemplate jdbcTemplate) {
 
   @WriteTransaction
   public void writeActivity(UUID workspaceId, DbWorkspaceActivityLog dbWorkspaceActivityLog) {
+    logger.info(
+        String.format(
+            "#writeActivity: workspaceId=%s, operationType=%s, actorEmail=%s, actorSubjectId=%s",
```

Comment:
What does the `#` do in the format string?

Reply:
nothing. it's just i generally like to have a # in front of a method to signify that this is from this method call from this class.